src/main/scala/bad/robot/temperature/server/StaticResources.scala

```diff
@@ -20,7 +20,7 @@ object StaticResources {
       resources(request)
 
     response.map {
-      case Successful(resp) => resp.putHeaders(`Cache-Control`(`no-cache`(), `no-store`, `max-age`(Zero)))
+      case Successful(resp) => resp.putHeaders(`Cache-Control`(`public`, `max-age`(Duration(31536000, SECONDS))))
```

Comment:
I might end up changing this to be `365 days` for clarity. The other thing I'm curious about is how clients work out genuine changes to the js or css, is that all handled 'for free'? i.e they will reload if they think they need to?

Reply:
+1 for changing to seconds to improve readability. Usually a hash or version number is embedded in the static resource name that forces clients to get the new version when it's referenced.